templates/chainbundle.crt.erb

```diff
@@ -0,0 +1 @@
+<%= @ssl_chainbundle_cert %>
```

Comment:
Just a small nitpick, we might want to add newlines to the end of these templates. Although things will work fine without them, it's POSIX standard to do this for all text files.
```suggestion
<%= @ssl_chainbundle_cert %>

```